### PR TITLE
GEOPY-472

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-472 - input file needs a validate flag that bypasses validations in ui_json setter.**

### CODE REVIEW

**Review by benk-mira — COMMENTED** [review]
Couple of things to discuss..

File: geoh5py/ui_json/utils.py [hunk 1]
```diff
@@ -140,10 +140,14 @@ def set_enabled(ui_json: dict, parameter: str, value: bool):
         parameters = find_all(group, "groupOptional")
         if parameters:
             is_group_optional = True
+            enabled_change = False
             for form in group.values():
+                enabled_change |= (
```
[the remark this answers is not in the document]
Comment (domfournier) [reply geoh5py/ui_json/utils.py]: Done

**Review by benk-mira — APPROVED** [review]
All good here

**Review by benk-mira — APPROVED** [review]
Approved!

**Pull request merged**